test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -5317,186 +5317,6 @@ BOOST_AUTO_TEST_CASE(event_signature_in_library)
 	)
 }
 
-BOOST_AUTO_TEST_CASE(abi_encode_with_selector)
-{
-	char const* sourceCode = R"(
-		contract C {
```

Comment:
So this v1 test has actually been running with v2 which is now the default? I guess we may have more cases like this in `SolidityEndToEndTest.cpp`?

Reply:
It was always running with both encoders. If we do not explicitly specify the encoder, it always tries both, doesn't it (at least in the soltest_all run).

Reply:
As long as we run both versions. But yeah, in CI we do so it's fine I guess.